index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
+<code>credentialSubject.id</code> is a Decentralized Identifier (DID).
+        </p>
+
+       <pre class="example nohighlight"
+          title="Usage of the confirmationMethod property of type DIDAuthWithSubjectIdConfirmation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    <span class="highlight">"confirmationMethod": {
+      "type": "DIDAuthWithSubjectIdConfirmation"
+    }</span>,
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  "proof": { <span class="comment">...</span> }
+}
+        </pre>
+
+        <p>
+The type <code>VerificationKeyConfirmation</code> defines that a
+<a>verifier</a> MAY validate the confirmation method by verifying the proof of
+the <a>verifiable presentation</a> with the verification material contained in
+<code>publicKeyJwk</code> or <code>publicKeyMultibase</code>.
+        </p>
+
+       <pre class="example nohighlight"
+          title="Usage of the confirmationMethod property of type VerificationKeyConfirmation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    <span class="highlight">"confirmationMethod": [{
+      "type": "VerificationKeyConfirmation",
+      "publicKeyJwk": {
+        "crv": "Ed25519",
+        "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ",
+        "kty": "OKP",
+        "kid": "_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A"
+      }
+    },{
+      "type": "VerificationKeyConfirmation",
+      "publicKeyMultibase": "zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+    }]</span>,
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  "proof": { <span class="comment">...</span> }
+}
+        </pre>
+
+        <p class="note">
+A confirmation method can express various metadata such as the <a>issuer</a>'s
+level of confidence that the <a>holder</a> is the entity referenced by a
+<a>subject</a> of the <a>verifiable credential</a>, specific form factors or
+mechanisms of authenticators, and/or references to other <a>verifiable credentials</a>
+or versioned trust frameworks. For example, an <a>issuer</a> can make a
+<a>claim</a> about a confirmation method that is based on a cryptographic key
+pair, but to produce a signature using that key, the <a>holder</a> has to unlock
+a device using multi-factor authentication.
```

Comment:
Just changing "confirmation method" to "confidence method" here makes it clear that it's the better word, IMO.

```suggestion
A confidence method can express various metadata such as the <a>issuer</a>'s
level of confidence that the <a>holder</a> is the entity referenced by a
<a>subject</a> of the <a>verifiable credential</a>, specific form factors or
mechanisms of authenticators, and/or references to other <a>verifiable credentials</a>
or versioned trust frameworks. For example, an <a>issuer</a> can make a
<a>claim</a> about a confidence method that is based on a cryptographic key
pair, but to produce a signature using that key, the <a>holder</a> has to unlock
a device using multi-factor authentication.
```